README.md

```diff
@@ -87,6 +87,10 @@ Rollbar::init($config, $set_exception_handler, $set_error_handler);
 ?>
 ```
 
+### For CodeIgniter Users
+
+If you are using CodeIgniter and wish to report on E_NOTICE level errors you'll have to be sure to run `Rollbar::init` from inside the Controller constructor.
```

Comment:
I cannot remember for the life of me. Only just now noticed this.

I think CodeIgniter was swallowing the E_NOTICE errors before Rollbar got a chance to see them. Since Rollbar behaves nicely (not swallowing) errors, sticking the init in the constructor meant that the Rollbar error handler got queued up *before* the CodeIgniter one.

I thought that in PHP all code was executed on every request by default anyway. (I'm pretty sure CodeIgniter is not a long-running app like Laravel). So yes. While it does get called on every request, it's not a terribly big deal.